### PR TITLE
Ss

### PR DESCRIPTION
**Pull request by Psyruss77**
### Why:

**Closes [issue link]**

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks for opening a PR 👍 Please fill out the [pull request template](https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) in the top comment in this pull request, then we'll get this up for review!

File: content/actions/guides/storing-workflow-data-as-artifacts.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ For example, your repository or a web application might contain SASS and TypeScr
 |   
 ```
 
-This example shows you how to create a workflow for a Node.js project that `builds` the code in the `src` directory and runs the tests in the `tests` directory. You can assume that running `npm test` produces a code coverage report named `code-coverage.html` stored in the `output/test/` directory.
```
Comment (Alvinchong1989) [inline content/actions/guides/storing-workflow-data-as-artifacts.md:77]:
```suggestion
This example shows you how to create a workflow for a Node.js project that `builds` the code in the `src` directory and runs the tests in the `tests` directory. You can assume that running `npm test` produces a code coverage report named `code-coverage.html` stored in the `output/test/` directory.
```

File: content/actions/guides/storing-workflow-data-as-artifacts.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ For example, your repository or a web application might contain SASS and TypeScr
...
+This example shows you how to create a workflow for a Node.js project that builds the code in the `src` directory and runs the tests in the `tests` directory. You can assume that running `npm test` produces a code coverage report named `code-coverage.html` stored in the `output/test/` directory.
```
Comment (Alvinchong1989) [inline content/actions/guides/storing-workflow-data-as-artifacts.md:77]:
```suggestion
This example shows you how to create a workflow for a Node.js project that builds the code in the `src` directory and runs the tests in the `tests` directory. You can assume that running `npm test` produces a code coverage report named `code-coverage.html` stored in the `output/test/` directory.
```

File: content/actions/guides/storing-workflow-data-as-artifacts.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ For example, your repository or a web application might contain SASS and TypeScr
...
 
 The workflow uploads the production artifacts in the `dist` directory, but excludes any markdown files. It also and uploads the `code-coverage.html` report as another artifact.
```
Comment (Alvinchong1989) [inline content/actions/guides/storing-workflow-data-as-artifacts.md:79]:
```suggestion
 The workflow uploads the production artifacts in the `dist` directory, but excludes any markdown files. It also and uploads the `code-coverage.html` report as another artifact.
```

File: content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md [hunk 4]
```diff
@@ -45,6 +45,10 @@ Before enabling {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data v
  ``` shell
 $ ghe-dep-graph-enable
 ```
+{% note %}
+**Note**: For more information about enabling access to the administrative shell via SSH, see "[Accessing the administrative shell (SSH)](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh)."
```
Comment (Alvinchong1989) [inline content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md:49]:
```suggestion
**Note**: For more information about enabling access to the administrative shell via SSH, see "[Accessing the administrative shell (SSH)](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh)."
```

File: content/actions/guides/storing-workflow-data-as-artifacts.md [hunk 5]
```diff
@@ -74,7 +74,7 @@ For example, your repository or a web application might contain SASS and TypeScr
 |   
 ```
```
Comment (Alvinchong1989) [inline content/actions/guides/storing-workflow-data-as-artifacts.md:75]:
```suggestion
 ```
```

File: content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md (hunk 4 above)
Comment (Alvinchong1989) [inline content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md:49]:
```suggestion
**Note**: For more information about enabling access to the administrative shell via SSH, see "[Accessing the administrative shell (SSH)](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh)."
```

File: content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md [hunk 6]
```diff
@@ -45,6 +45,10 @@ Before enabling {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data v
...
+{% endnote %}
```
Comment (Alvinchong1989) [inline content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md:50]:
```suggestion
{% endnote %}
```

File: content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md [hunk 7]
```diff
@@ -45,6 +45,10 @@ Before enabling {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data v
...
+
 3. Return to {% data variables.product.prodname_ghe_server %}.
```
Comment (Alvinchong1989) [inline content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md:48]:
```suggestion
 3. Return to {% data variables.product.prodname_ghe_server %}.
```

**Pull request closed without merging**